composer.json

```diff
@@ -20,76 +20,77 @@
-        "doctrine/common": "~2.5",
-        "doctrine/data-fixtures": "~1.1.0",
-        "doctrine/doctrine-bundle": "~1.3",
-        "doctrine/doctrine-cache-bundle": "~1.0",
-        "doctrine/doctrine-fixtures-bundle": "~2.2",
-        "doctrine/doctrine-migrations-bundle": "~1.0",
-        "doctrine/orm": "^2.4.8",
-        "friendsofsymfony/oauth-server-bundle": "~1.4",
-        "friendsofsymfony/rest-bundle": "~1.5",
-        "fzaninotto/faker": "~1.2",
+
+        "doctrine/collections": "^1.3",
+        "doctrine/data-fixtures": "^1.1",
+        "doctrine/doctrine-bundle": "^1.6",
+        "doctrine/doctrine-cache-bundle": "^1.3",
+        "doctrine/doctrine-fixtures-bundle": "^2.3",
+        "doctrine/doctrine-migrations-bundle": "^1.2",
+        "doctrine/orm": "^2.5",
+        "friendsofsymfony/oauth-server-bundle": "^1.5",
+        "friendsofsymfony/rest-bundle": "^1.8",
+        "fzaninotto/faker": "^1.6",
         "gedmo/doctrine-extensions": "^2.4.12",
-        "incenteev/composer-parameter-handler": "~2.0",
-        "jms/serializer-bundle": "^0.13|^1.0",
-        "knplabs/knp-gaufrette-bundle": "~0.1",
+        "incenteev/composer-parameter-handler": "^2.1",
+        "jms/serializer-bundle": "^1.1",
+        "knplabs/knp-gaufrette-bundle": "^0.3",
         "knplabs/knp-menu-bundle": "^2.1",
-        "liip/imagine-bundle": "~1.0",
+        "liip/imagine-bundle": "^1.6",
         "ocramius/proxy-manager": "^1.0",
-        "payum/payum": "~1.3",
-        "payum/payum-bundle": "~2.1",
-        "php-http/guzzle6-adapter": "~1.1",
+        "payum/payum": "^1.3",
+        "payum/payum-bundle": "^2.1",
+        "php-http/guzzle6-adapter": "^1.1",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
-        "sensio/distribution-bundle": "~3.0",
+        "sensio/distribution-bundle": "^5.0",
         "sonata-project/intl-bundle": "^2.2",
-        "stof/doctrine-extensions-bundle": "~1.1",
-        "swiftmailer/swiftmailer": "~5.0",
-        "symfony/monolog-bundle": "^2.8",
-        "symfony/polyfill-iconv": "~1.0",
-        "symfony/polyfill-intl-icu": "~1.0",
-        "symfony/polyfill-mbstring": "~1.0",
-        "symfony/swiftmailer-bundle": "^2.3",
-        "symfony/symfony": "^2.8",
-        "twig/extensions": "~1.0",
-        "twig/twig": "~1.11",
-        "webmozart/assert": "^1.0",
-        "white-october/pagerfanta-bundle": "~1.0",
-        "willdurand/hateoas-bundle": "^0.4|^1.0",
-        "winzou/state-machine-bundle": "^0.2|^0.3",
-        "zendframework/zend-hydrator": "^1.0|^2.0",
-        "zendframework/zend-stdlib": "^2.0|^3.0"
+        "stof/doctrine-extensions-bundle": "^1.2",
+        "swiftmailer/swiftmailer": "^5.4",
+        "symfony/monolog-bundle": "^3.0",
+        "symfony/polyfill-iconv": "^1.3",
+        "symfony/polyfill-intl-icu": "^1.3",
+        "symfony/polyfill-mbstring": "^1.3",
+        "symfony/swiftmailer-bundle": "^2.4",
+        "symfony/symfony": "^3.2@rc",
```

Comment:
Stable version will be released this week, so I think it's worth to aim for Symfony 3.2